zap-db.yml

```diff
@@ -5,13 +5,13 @@
 # 1) Deploy the middleware again. This creates the middleware and gateway databases
 # 2) Run the database migrations to create the required tables on one of the app servers:
 #    $ cd /opt/www/middleware.<env>
-#    $ php app/console middleware:migrations:migrate --env=prod
+#    $ php app/console middleware:migrations:migrate --env:prod
 #    $ cd /opt/www/gateway.<env>
```

Comment:
"--env=prod" is correct IMO. Not sure about "--env:prod"

Reply:
Replaced this entry by mistake.